meshtastic/__init__.py

```
@@ -232,7 +236,35 @@ def _sanitize_last_received(as_dict: dict[str, Any]) -> dict[str, Any]:
+                    else REDACTED_TEXT
+                )
+            admin_sanitized["raw"] = raw_sanitized_dict
+        elif hasattr(raw_admin, "session_passkey"):
+            try:
+                raw_sanitized_obj: Any = copy.deepcopy(raw_admin)
+            except Exception:  # noqa: BLE001 - preserve packet flow on odd payloads
+                raw_sanitized_obj = raw_admin
+            try:
+                raw_sanitized_obj.session_passkey = REDACTED_BYTES
+            except Exception:  # noqa: BLE001 - best effort redaction only
+                pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)